### ISSUE TITLE
ioctl version command show unknown version info 

### ISSUE THREAD

**Issue by envestcc** [iotexproject/iotex-core#3768]
### What version of iotex-core image (or code branch) are you using?
Download from [ioctl-darwin-amd64](https://github.com/iotexproject/iotex-core/releases/download/v1.9.1/ioctl-darwin-amd64)

### What operating system and processor architecture are you using?
macOS Monterey 12.5.1
Apple M1 Pro

### What did you do? If possible, provide a recipe for reproducing the error.
run shell command
```
./Downloads/ioctl-darwin-amd64 version
```

### What did you expect to see?
Should show the v1.9.1 version info

### What did you see instead?
```
Client:
packageVersion:"NoBuildInfo" packageCommitID:"NoBuildInfo" gitStatus:"NoBuildInfo" goVersion:"NoBuildInfo" buildTime:"NoBuildInfo"

api.mainnet.iotex.one:443:
packageVersion:"v1.9.1" packageCommitID:"83008edb897360de2efbbc6097821a5bd9b4fd54" gitStatus:"clean" goVersion:"go version go1.18.5 linux/amd64" buildTime:"2023-01-05-UTC/06:33:11"

```

**Comment by dalaohuniu** [comment]
thankyou

**Comment by hunshenshi** [comment]
have fixed in v1.10+
